src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -89,57 +89,60 @@ interface DropdownState {
 
 export interface DropdownProps extends StatusTextCommonProps {
   /**
-   * Unique id
+   * HTML id attribute
    * If no id is specified, one will be generated automatically
    */
   id?: string;
-  /** Name used for input's form value. */
+  /** HTML name attribute. */
   name?: string;
-  /** Default for non controlled Dropdown, intially selected value */
+  /** Sets a default, initially selected value for non controlled Dropdown */
   defaultValue?: string;
-  /** Controlled selected value, overrides defaultValue if provided. */
+  /** Controlled selected value, overrides `defaultValue` if provided. */
   value?: string;
   /** Label for the Dropdown component. */
   labelText: ReactNode;
   /** Hint text to be shown below the label */
   hintText?: string;
-  /** Visual hint to show if nothing is selected and no value or defaultValue is provided */
+  /** Visual hint to show if nothing is selected and no value or defaultValue is provided. Do not use for instructions. */
```

Comment:
Should this also have short clarification why it should not be used for instructions, as it is a bit vague instruction otherwise? 

"Should not be used for instructions, ..." might also suit tone of instructions better?

Reply:
This is now addressed in the latest commit